.gitignore

```diff
@@ -3,4 +3,5 @@ __pycache__
 molpipeline.egg-info/
 lib/
 build/
+lightning_logs/
```

Comment:
I also thought about this while reading the PR. It will not hurt much but is only relevant for the extras and not the core library. I think I am fine with keeping it.